requirements-test.txt

```diff
@@ -2,3 +2,5 @@ coverage
 flake8
 pytest
 tox
+sphinx
```

Comment:
On May 18, 2015 7:37:32 PM CDT, Cameron Paul <notifications@github.com> wrote:
>> @@ -2,3 +2,5 @@ coverage
>>  flake8
>>  pytest
>>  tox
>> +sphinx
>
>Why is sphinx needed for tests?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/photoshell/rawkit/pull/13/files#r30560405

Travis builds the docs now to make sure we don't break them.
-- 
Sam Whited
pub 4096R/54083AE104EA7AD3